projects/packages/search/src/classic-search/class-classic-search.php

```diff
@@ -1564,6 +1590,26 @@ public function get_filters( WP_Query $query = null ) {
 
 						break;
 
+					// The filter `author` is NOT supported in Classic Search. This is used to be compatible for filters in the Footer section with Instant Search.
+					case 'author':
+						$name = preg_split( '/\/(.+)/', $item['key'] )[0];
```

Comment:
Is it possible that `preg_split` returns `false` rather than an array, which would then throw a warning?